PetaPoco.Tests.Integration/Models/OrderLine.cs

```diff
@@ -25,7 +25,10 @@ public class OrderLine
         public decimal SellPrice { get; set; }
 
         [ResultColumn]
-        public decimal Total { get; set; }
```

Comment:
@LankyPaddy I need to confirm this isn't being used by other tests. Looks good so far